installers/conf/config_sync.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Copyright 2020 Crunchy Data Solutions, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [ -z ${PGOROOT} ]
+then
+	echo "error: \$PGOROOT must be set"
+	exit 1
+fi
+
+INSTALLER_DIR="$PGOROOT/installers"
+CONF_DIR="$INSTALLER_DIR/conf"
+TEMPLATE_DIR="$CONF_DIR/templates"
+
+if [[ -d ${INSTALLER_DIR} || -d ${CONF_DIR} || -d ${TEMPLATE_DIR} ]]
```

Comment:
Thinking we should move this script to the `hack` directory under the project root, alongside the `update-codegen.sh` and `verify-codegen.sh` scripts.  This is a pattern that emerged within the Kube world for storing various developer-facing scripts, and this appears to fall into that category:

https://github.com/kubernetes/kubernetes/tree/master/hack
https://github.com/kubernetes/kops/blob/master/docs/development/hack.md
https://github.com/kubernetes/kops/issues/444

This will allow us to continue consolidating our various dev scripts into a single, central location.

Reply:
Yeah that sounds like a good idea, I can move that script. With that change, and moving the `helm_template.yaml` into the helm directory, the only thing that is in the `installers/conf` directory is the master `values.yaml`. We can move that file and not have to add a new directory. What would yall think about one of these:

- using the values.yaml in the ansible directory as the master
- moving the master values.yaml to the root installers directory, so `installers/values.yaml`

Reply:
I personally like the idea of just using Ansible as the master.  If Ansible is essentially always the "master" config anyway (i.e. they are always identical), then seems to make sense to just use it as the master, instead of having a duplicate copy.

And this also further consolidates around Ansible essentially being the master installer for everything (i.e. master install engine as used by `kubectl` and Helm, the master config location, etc.).

Reply:
Sounds good to me, I'll move forward with that change